src/containers/SubjectPage/components/MovedTopicPage.tsx

```diff
@@ -60,17 +63,16 @@ interface Props {
 const Wrapper = styled("div", {
   base: {
     display: "flex",
+    gap: "xxlarge",
     flexDirection: "column",
-    alignItems: "flex-start",
-    gap: "xsmall",
-    paddingBlock: "medium",
   },
 });
 
 const SearchResultListWrapper = styled("ul", {
   base: {
     display: "flex",
-    flexDirection: "column",
+    justifyContent: "center",
+    flexWrap: "wrap",
```

Comment:
Kanskje dette kunne vært en grid istedenfor? Da kan kortene auto-skalere

Reply:
Og ha breakpoints for 1,2 og 3 kort i bredden?

Reply:
Tror faktisk det går fint jeg.

Reply:
Jeg tror at denne siden så og si aldri skal vises så det tror jeg og. Men om vi liker det bedre så kan jeg endre altså :smiley_cat: